x/blob/README.md

```diff
@@ -23,10 +23,6 @@ When a `MsgPayForBlob` is processed, it consumes gas based on the blob size.
 
 - [`MsgPayForBlob`](https://github.com/celestiaorg/celestia-app/blob/8b9c4c9d13fe0ccb6ea936cc26dee3f52b6f6129/proto/blob/tx.proto#L39-L44) pays for the blob to be included in the block.
 
-## PrepareProposal
-
-The malleation process occurs during the PrepareProposal step.
```

Comment:
I would still have a section here talking about how the PFB gets put in a wrapper that also contains the index(es) of the blob(s) that it references

Reply:
proposal to tackle this in a stand-alone PR b/c there are several things in this file that need updating and they're not all related to malleation: https://github.com/celestiaorg/celestia-app/issues/1268